actor-testkit-typed/src/main/scala/org/apache/pekko/actor/testkit/typed/internal/TestProbeImpl.scala

```diff
@@ -43,7 +43,7 @@ import pekko.actor.typed.scaladsl.Behaviors
 import pekko.annotation.InternalApi
 import pekko.japi.function.Creator
 import pekko.util.BoxedType
-import pekko.util.JavaDurationConverters._
+import scala.jdk.DurationConverters._
```

Comment:
A few imports need to move to keep them with the Scala imports

Reply:
I think we should do that sort in a dedicated PR, with `sortImports`.